releases/release_notes_09_2022.md

```diff
@@ -293,5 +293,100 @@ In case the user wants to run the workload on a TDX capable hardware, using QEMU
 
 * *TBD: instructions to build encrypted container image and other requirements (attestation, key etc)* *
 
+## Experience trusted ephemeral storage for container image and RW layer
```

Comment:
Could we change this to a single "#" and not "##" so this would be a separated section then the 4 use cases?
Basically a new section that talks about the trusted ephemeral storage

Reply:
And maybe rename it `Trusted Ephemeral Storage`

Reply:
Thanks @sameo @ariel-adam ,resolved now